### ISSUE TITLE
photo uploads  ?? where? what?

### ISSUE THREAD

**Issue by treyfenton** [nitinhayaran/Justified.js#3]
Hey there,

I like the plugin, however i'm a bit confused.

So in my code I call

<script src="jquery.justified.images.js"></script>

But there's no "jquery.justified.images.js" script in the given docs.

Then you say to set my images in the array 'images'... 
Am I to create the array myself? And am I to put that into/save as "jquery.justified.images.js"?

Just confused as to where and how the images populate.
Basically, where do I put the source for the images so that this thing will work?

Any and all help is appreciated!
Thanks!

**Comment by buildingawesomethings** [comment]
Hi,

First of all, really awesome work you did here!
I'm having the same difficulties though, i tried to make sense of the JSON callback from the Flickr API and trying to reproduce a similar array with my own local images but without any luck.

Can you maybe provide us with a simple example of some basic HTML structure with hardcoded image links instead of the Flickr json response?

Hope to hear from you :)

Kind regards

**Comment by ValeriaMachado** [comment]
var photos = ["https://images.unsplash.com/photo-1521913626209-0fbf68f4c4b1?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80"]
